### DISCUSSION TITLE
Add 2FA

### DISCUSSION THREAD

**Discussion by sreenath87** [angristan/openvpn-install#866] [Ideas]
Would it be possible for you to include 2FA using TOTP to the script you have. (google Authenticator) ?

Thank you for your efforts

**Comment by ysfnsrv** [comment]
> Would it be possible for you to include 2FA using TOTP to the script you have. (google Authenticator) ?
> 
> Thank you for your efforts
on this version it works 
root@vpn:~# openvpn --version
OpenVPN 2.6.12 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] [DCO]
library versions: OpenSSL 3.0.13 30 Jan 2024, LZO 2.10

https://blog.gordonbuchan.com/blog/index.php/2022/12/26/integrating-open-source-software-in-the-enterprise-chapter-2-using-linux-and-openvpn-to-create-a-virtual-private-network-vpn-server-with-two-factor-authentication-2fa-enabled-using-google-authent/
